features/response/response.feature

```diff
@@ -158,3 +158,29 @@ Feature: Handling responses from the ID Broker API
     Given a call to "mfaVerify" will return a 400 response
     When I call mfaVerify with the necessary data
     Then the result should be false
+
+  Scenario: Handling a successful createMethod call
+    Given a call to "createMethod" will return a 200 response
+    When I call createMethod with the necessary data
+    Then an exception should NOT have been thrown
+    And the result should be an array
+
+  Scenario: Handling a rate-limited call to verifyMethod
+    Given a call to "verifyMethod" will return a 429 response
+    When I call verifyMethod with the necessary data
+    Then a Method rate-limit exception SHOULD have been thrown
+
+  Scenario: Handling a "correct" response from verifyMethod
+    Given a call to "verifyMethod" will return a 200 response
+    When I call verifyMethod with the necessary data
+    Then the result should be an array
+
+  Scenario: Handling a "wrong" response from verifyMethod
+    Given a call to "verifyMethod" will return a 400 response
+    When I call verifyMethod with the necessary data
+    Then the Method verify exception SHOULD have been thrown
+
+  Scenario: Handling a "wrong" response from resendMethod
+    Given a call to "resendMethod" will return a 400 response
+    When I call resendMethod with the necessary data
+    Then the result should be false
```

Comment:
Out of curiosity, why do we have the resendMethod return `false` (rather than throw a custom exception) on a `400` response?

Reply:
No good reason. Thanks for catching this.